source/operations/cert-manager.rst

```diff
@@ -0,0 +1,108 @@
+.. _minio-certmanager:
+
+============
+cert-manager
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+TLS certificate management with cert-manager
+--------------------------------------------
+
+This guide shows you how to install `cert-manager <https://cert-manager.io>`__ and set up the global ``Cluster Issuer``.
+
+cert-manager manages certificates within Kubernetes clusters.
+The MinIO Operator supports using cert-manager for certificates as an alternative to the MinIO Operator managing certificates for itself and its tenants.
+
+cert-manager obtains valid certificates from an ``Issuer`` or ``ClusterIssuer`` and can automatically renew certificates prior to expiration.
+
+A ``ClusterIssuer`` issues certificates for multiple namespaces.
+An ``Issuer`` only mints certificates for its own namespace.
+
+The following graphic depicts how various namespaces make use of either an ``Issuer`` or ``ClusterIssuer`` type.
+
+- cert-manager is installed in the ``cert-manager`` namespace, which does not have either an ``issuer`` or a ``ClusterIssuer``.
+- The ``default`` namespace receives the global ``Cluster Issuer``.
+- Each tenant's namespace receives a local ``Issuer``.
+- The ``minio-operator`` namespace receives a local ``Issuer``. 
+
+.. image:: /images/k8s/cert-manager-cluster.svg
+   :width: 600px
+   :alt: A graphic depiction of a kubernetes cluster with five separate namespaces represented by different boxes. One is labeled minio-operator with a text box that says "minio-operator: issuer". A second is labeled tenant-1 with text box that says "tenant-1: issuer". A third is labeled default with a text box that says "root:ClusterIssuer". A fourth is labeled tenant-2 with a text box that says "tenant-2: issuer". The fifth is labeled cert-manager with a text box that says "minio-operator: issuer".
```

Comment:
An idea for the alt-text
```suggestion
   :alt: A Kubernetes cluster with five namespaces, shown as a box for each namespace in the cluster. The minio-operator namespace contains a "minio-operator: issuer" local issuer. The default namespace contains a "root: ClusterIssuer" cluster issuer. The cert-manager namespace contains a "minio-operator: issuer" local issuer. The remaining two namespaces are individual tenants, "tenant-1" and "tenant-2", each with its own local issuer.
```
(I'm not sure how screen readers handle quotes and terminal punctuation, so I followed your style.)

Reply:
Alt text is not my forte, so I'm open to whatever suggestions. I just took what you suggested.

Reply:
Warning, opinionated digression ahead:

My take on alt-text is "What are the important ideas the author intended people to get from this image?" And everybody has their own opinion on how it works for them and their technology. For me, reading a description of every detail (on my tiny communications device) makes it harder to follow the purpose of the image.

Lately I've been seeing more literal image descriptions and I wonder if they are auto generated. For example, computers can tell you the image has overlapping circles containing text, but perhaps not that it's a Venn diagram illustrating a joke.